source/includes/documentation/_vue_js_elements.md

```diff
@@ -1548,47 +1548,243 @@ and [JDI vuetify page tests for windows](https://github.com/jdi-testing/jdi-ligh
 
 ### 5.14 Lists
 
-[Vuetify documentation page](https://vuetifyjs.com/en/components/lists/)
```

Comment:
Please consider to make part about Tests less extensive - find few examples that is most interesting and valuable from your pain of view.

Reply:
Only two tests examples are left now.

Reply:
I decreased amount of tests examples